demo/SnowplowSwiftSPMDemo/Dependencies/Package.swift

```diff
@@ -13,9 +13,9 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SnowplowTracker", url: "https://github.com/snowplow/snowplow-objc-tracker", .branch("master")),
+        // .package(name: "SnowplowTracker", url: "https://github.com/snowplow/snowplow-objc-tracker", .branch("master")),
         // .package(name: "SnowplowTracker", path: "../../../../snowplow-objc-tracker"),
-        // .package(name: "SnowplowTracker", path: "../../../.."),
+        .package(name: "SnowplowTracker", path: "../../../.."),
     ],
```

Comment:
How do you deal with this if the trackers github actions need this to be merged but this depends on the tracker being merged to build?

Reply:
It's something that needs improvements.
Cocoapods and Carthage dependencies are set by GH Actions script, so for them the demo being built on the local tracker under testing.
For SPM, this line 18 specifies the path of the tracker under testing. So It's not needed that the tracker has been merged to build.
Unfortunately, it's possible there are edge cases where this solution wouldn't work perfectly. It's something to consider going forward.